src/app/add-todo/add-todo.component.html

```diff
@@ -0,0 +1,12 @@
+<h3>
+  Add new todo:
+</h3>
+
+<form>
+  <div class="form-group">
+    <label for="todoDescription">Todo Description</label>
+    <br>
+    <input type="text" [(ngModel)]="newTodo.description" placeholder="description" name="description" />
+  </div>
+  <button (click)="add(newTodo)" type="submit" class="btn btn-primary">Add new todo</button>
```

Comment:
Nije potrebno prosledjivanje newTodo kao atributa, on ti je vec dostupan u typescriptu kao this.newTodo